elements/lisk-chain/src/utils.ts

```diff
@@ -27,3 +27,13 @@ export const getId = (blockBytes: Buffer): string => {
 
 	return id;
 };
+
+export const uniqBy = (
```

Comment:
I think instead of making it generic, we can just check in each case. Mostly checking if the id or address exists in the cache, it uses it

Reply:
Decision to keep `uniqBy` due to efficiency improvement.